akka-typed-tests/src/test/scala/docs/akka/typed/IntroSpec.scala

```diff
@@ -128,13 +128,8 @@ class IntroSpec extends TypedSpec {
         Immutable[akka.NotUsed] {
           (_, _) ⇒ Unhandled
         } onSignal {
```

Comment:
while we are at this topic, should we use above infix notation or ordinary `.onSignal {`
I prefer infix only for symbolic operators and short names, i.e. my vote is `.onSignal {`